testutils/src/main/scala/za/co/absa/enceladus/testutils/e2eSparkRunner/E2ESparkRunner.scala

```diff
@@ -0,0 +1,72 @@
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.testutils.e2eSparkRunner
+
+import org.apache.log4j.{LogManager, Logger}
+import za.co.absa.enceladus.testutils.HelperFunctions
+import za.co.absa.enceladus.testutils.models.ProcessMeasurement
+
+import sys.process._
+import scala.io.Source
+
+object E2ESparkRunner {
+  private val log: Logger = LogManager.getLogger(this.getClass)
+
+  private val stdJarPath: String = "$DCE_JAR_PATH/$DCE_STD_SPT_JAR"
+  private val confJarPath: String = "$DCE_JAR_PATH/$DCE_CNFRM_SPT_JAR"
+  private val testUtilsJarPath: String = "$DCE_JAR_PATH/$TEST_UTILS_JAR"
+
+  private val stdClass = "--class za.co.absa.enceladus.standardization.StandardizationJob"
+  private val confClass = "--class za.co.absa.enceladus.conformance.DynamicConformanceJob"
+  private val compClass = "--class za.co.absa.enceladus.testutils.datasetComparison.ComparisonJob"
+
+  private def runBashCmd(bashCmd: String): String = {
+    (s"echo $bashCmd" #| "bash").!!
+  }
+
+  def main(args: Array[String]): Unit = {
+    val cmd = CmdConfig.getCmdLineArguments(args)
+
+    val standartisation = s"spark-submit ${cmd.sparkConf} $stdClass $stdJarPath ${cmd.standartizationJobConf}"
+    log.debug(standartisation)
+    log.info("Running Standartization")
+    val ProcessMeasurement(stdTime, standartisationRes) = HelperFunctions.calculateTime { runBashCmd(standartisation) }
+    log.debug(standartisationRes)
+    log.info("Standartization Passed")
+    val stdComparison = s"spark-submit ${cmd.sparkConf} $compClass $testUtilsJarPath ${cmd.stdComparisonConf}"
+    log.debug(stdComparison)
+    log.info("Running Standartization Comparison")
+    val stdComparisonRes: String = runBashCmd(stdComparison)
+    log.debug(stdComparisonRes)
+    log.info("Standartization Comparison Passed")
+    val conformance = s"spark-submit ${cmd.sparkConf} $confClass $confJarPath ${cmd.conformanceJobConf}"
+
+    log.debug(conformance)
+    log.info("Running Conformance")
+    val ProcessMeasurement(confTime, conformanceRes) = HelperFunctions.calculateTime { runBashCmd(conformance) }
+    log.debug(conformanceRes)
+    log.info("Conformance Passed")
+    val confComparison = s"spark-submit ${cmd.sparkConf} $compClass $testUtilsJarPath ${cmd.confComparisonConf}"
+    log.debug(confComparison)
+    log.info("Running Conformance Comparison")
+    val confComparisonRes: String = runBashCmd(confComparison)
+    log.debug(confComparisonRes)
+    log.info("Conformance Comparison Passed")
+
+    log.info(s"Standartization and Conformance passed. It took them $stdTime and $confTime respectively")
```

Comment:
The time is in seconds, Right?

Reply:
True. I will format it.

Reply:
I changed the formatting. Now it shows minutes and seconds in a human readable format